tests/factories.py

```diff
@@ -369,3 +369,11 @@ class ScheduleAByStateRecipientTotalsFactory(BaseFactory):
     class Meta:
         model = models.ScheduleAByStateRecipientTotals
     idx = factory.Sequence(lambda n: n)
+
+class AuditCaseFactory(BaseFactory):
+    class Meta:
+        model = models.AuditCase
+    idx = factory.Sequence(lambda n: n)
+    # audit_case_id = factory.Sequence(lambda n: n)
+    # primary_category_id = -1
+    # sub_category_id = -2
```

Comment:
There's no need to keep old commented lines, please remove.  If we need to retrieve them we can pull them out of the history in Git. :-)